libqtile/widget/keyboardlayout.py

```diff
@@ -39,8 +39,8 @@ class KeyboardLayout(base.InLoopPollText):
     and bind function "next_keyboard" to specific keys in order to change layouts.
 
     For example:
-        Key([mod], "space", lazy.widget["keyboardlayout"].next_keyboard(), desc="Next keyboard layout."),
 
+        Key([mod], "space", lazy.widget["keyboardlayout"].next_keyboard(), desc="Next keyboard layout."),
```

Comment:
Seems like you can drop this from the diff?

Reply:
That diff actually puts a new line between:
```
For example:
```
and
```python
Key([mod], “space”, lazy.widget[“keyboardlayout”].next_keyboard(), desc=”Next keyboard layout.”),
```

---

If you go to the [widget documentation](http://docs.qtile.org/en/latest/manual/ref/widgets.html#keyboardlayout) you will see that
```python
Key([mod], “space”, lazy.widget[“keyboardlayout”].next_keyboard(), desc=”Next keyboard layout.”),
```
is not treated as a code block. That is why I did this.
Diff looks kind of weird because I actually borrowed a new line from one below, which isn't showed here.

Reply:
Ok, can you put this in a separate patch?

Reply:
@tych0 Sure, I moved it to a separate commit.